sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -190,12 +190,12 @@ class CategorizedEntity(DictMixin):
     :type category: str
     :param subcategory: Entity subcategory, such as Age/Year/TimeRange etc
     :type subcategory: str
-    :param offset: Start position (in Unicode characters) for the
+    :param grapheme_offset: Start position (in Unicode characters) for the
```

Comment:
> in Unicode characters

I'm still wrapping my head around graphemes. Is this description accurate?

Reply:
This description came straight from their swagger... so I hope its accurate. Also still wrapping my head around it.

Reply:
Unicode "characters" could be confused with code-points, which are different from graphemes.

Reply:
According to [unicode.org](https://unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries)
```
To avoid ambiguity with the computer use of the term character, this is called a user-perceived character. [..] These user-perceived characters are approximated by what is called a grapheme cluster, which can be determined programmatically.
```
We could:
- Change it to `Grapheme starting point for the entity text`
- or change it to `(in user-perceived characters)` 
- or remove the `(in Unicode characters)`